app/dao/notifications_dao.py

```diff
@@ -455,6 +455,12 @@ def dao_get_notifications_by_to_field(service_id, search_term, notification_type
     else:
         raise InvalidRequest("Only email and SMS can use search by recipient", 400)
 
+    for special_character in {'_', '%', '/'}:
+        normalised = normalised.replace(
+            special_character,
+            '\\{}'.format(special_character)
```

Comment:
should this be a raw string? eg `r'\\_'` - or python will convert `\\` into a single backslash

Reply:
Single backslash is what I want – maybe would be clearer as `r'\{}'`?

Reply:
9103ca5

Reply:
ah i think i got myself confused by reading something from the postgres mailing list from 2000.

`r` is marginally clearer  but not enough for me to mind